lib/task.js

```diff
@@ -1,3 +1,4 @@
+lintError;
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>'lintError' is not defined.